api/webrtc/webrtc.go

```diff
@@ -99,8 +100,11 @@ func (c *Peer) Offer(label string) error {
 	if err != nil {
 		return err
 	}
+
+	c.OnOfferConnected = make(chan struct{}, 1)
```

Comment:
need to move this to constructer because very often people use it this way:
```
peer := NewPeer(...)
go func() { ... <-peer.OnOfferConnected ... }
peer.Offer()
```
so OnOfferConnected needs to be created when creating peer

Reply:
Yes, modified.